src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -34,10 +36,60 @@
 from .checks import check_config
 from .client import get_catalogi_client, get_documents_client, get_zaken_client
 from .models import ZGWApiGroupConfig, ZgwConfig
+from .utils import process_according_to_eigenschap_format
 
 logger = logging.getLogger(__name__)
 
 
+class VariablesProperties(TypedDict):
+    component_key: str
+    eigenschap: str
+
+
+def get_property_mappings_from_submission(
+    submission: Submission, connections: list[VariablesProperties]
+) -> dict[str, Any]:
+    """
+    Extract the values from the submission and map onto the provided properties (eigenschappen).
+    """
+    property_mappings = {}
+
+    # dict of {componentKey: eigenschap} mapping
+    simple_mappings = {
+        conn["component_key"]: conn["eigenschap"] for conn in connections
+    }
+
+    merged_data = dict(
+        submission.submissionvaluevariable_set.filter(
+            key__in=[conn["component_key"] for conn in connections]
+        ).values_list("key", "value")
+    )
+
+    for key, form_value in merged_data.items():
+        if key not in simple_mappings:
```

Comment:
this can never be `True` because the the keys of `simple_mapping` are used as a query `filter` clause, so the check is not needed. There's also no test coverage for that code and it's not possible to get coverage, I believe.